services/requester/requester.go

```diff
@@ -149,6 +146,13 @@ func (e *EVM) SendRawTransaction(ctx context.Context, data []byte) (common.Hash,
 		return common.Hash{}, err
 	}
 
+	if tx.GasPrice().Cmp(e.config.GasPrice) == -1 {
```

Comment:
A better convention is to compare with `0`, this way you can express with the sign itself if the first value is bigger or smaller.
```suggestion
	if tx.GasPrice().Cmp(e.config.GasPrice) < 0 {
```

Reply:
Oh, nice :raised_hands: